ddl/ddl_api.go

```diff
@@ -5210,7 +5210,7 @@ func checkPlacementLabelConstraint(label string) (placement.LabelConstraint, err
 	return r, nil
 }
 
-func checkPlacementLabelConstraints(rule *placement.Rule, labels []string) error {
+func checkPlacementLabelConstraints(rule *placement.RuleOp, labels []string) error {
```

Comment:
Placement-relative util functions and constants can also be moved into `ddl/placement` package.